### ISSUE TITLE
Install theme from subfolder

### ISSUE THREAD

**Issue by mandrasch** [SayHelloGmbH/git-installer#28]
Hi,

just tried installing a child theme within a subfolder.

Version used: https://wp.nico.dev/wp-content/uploads/2022/10/git-installer-0-2-0.zip

```
Repo: https://github.com/mandrasch/blog-childtheme
Branch: main
Directory: wp-content/themes/mandrasch-child/
Error: No valid WordPress Theme (style.css with "Theme Name" header) or WordPress Plugin (Plugin PHP file with "Plugin Name" header) was found in the wp-content/themes/mandrasch-child/ folder of the repository.
```
<img width="672" alt="image" src="https://user-images.githubusercontent.com/777278/200164049-71843196-0516-4dd5-887f-537259439316.png">

The tree which is checked:

https://api.github.com/repos/mandrasch/blog-childtheme/git/trees/main?recursive=1

I just briefly checked https://github.com/SayHelloGmbH/git-installer/blob/main/src/Package/provider/Github.php#L98, guess `$folder` is not considered in the comparison for style.css yet? 

```php
function ($element) use ($folder) {
                    if ($element['type'] !== 'blob') return false;
                    if (!str_starts_with($element['path'], $folder)) return false;
                    if ($element['path'] === 'style.css') return true;
                    $relativePath = substr($element['path'], strlen($folder));
                    if (str_contains($relativePath, '/')) return false;
                    return str_ends_with($relativePath, '.php');
}
```

If i add a second compare with $folder, installation works fine.

```php
if ($element['path'] === 'style.css') return true;
if ($element['path'] === $folder.'style.css') return true;
```

But not sure if `$relativePath = ...` is supposed to handle this. 

Thanks for providing this openly here, cheers!

**Comment by nico-martin** [comment]
Hi @mandrasch,
Thanks for the testing and the hint. I think I fixed it with https://github.com/SayHelloGmbH/git-installer/commit/b4651ce5679a4dab13e405b2ac52a3d98e5e8112
Could you try the new version to verify if that helped?
https://wp.nico.dev/wp-content/uploads/2022/11/git-installer-0-2-1-1667807205975.zip

**Comment by mandrasch** [comment]
> Hi @mandrasch, Thanks for the testing and the hint. I think I fixed it with [b4651ce](https://github.com/SayHelloGmbH/git-installer/commit/b4651ce5679a4dab13e405b2ac52a3d98e5e8112) Could you try the new version to verify if that helped? https://wp.nico.dev/wp-content/uploads/2022/11/git-installer-0-2-1-1667815885476.zip

Yes, worked well with this new version 👍
